OMCompiler/Compiler/FrontEnd/ModelicaBuiltin.mo

```diff
@@ -4171,6 +4171,20 @@ annotation(
 </html>"), preferredView="text");
 end generateScriptingAPI;
 
+// OMSimulator API calls
+function oms_getVersion "Returns the version of the OMSimulator."
+  output String version;
+external "builtin";
+annotation(preferredView="text");
+end oms_getVersion;
+
+function oms_setLogFile "sets the LogFilename defined by user."
+  input String filename;
+  output Integer status;
```

Comment:
It would be nice to add here what does the different numbers of the status means. See `oms_status`.

Reply:
If it's an enumeration, it is handled correctly in external C. Just a bit annoying to propagate to the scripting interface.